demo/fake_server.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Wanda.Executions.FakeServer do
```

Comment:
Do we need a `demo/fake_server.ex` file?
Why not simply put the `FakeServer` module in the `executions` folder

Reply:
Since FakeServer imports `Wanda.Factory` and `Wanda.Factory` is only available while testing, we need to play a bit with the paths in mix.exs to avoid always importing the Factory when it's not needed but allow it to be there when we are on the demo env. See `mix.exs`:

```
  # Specifies which paths to compile per environment.
  defp elixirc_paths(:test),
    do: [
      "lib",
      "test/support"
    ]

  defp elixirc_paths(:demo),
    do: [
      "test/support/factory.ex",
      "demo/fake_server.ex",
      "lib"
    ]
 ```

Reply:
Ok, I see